src/controllers/auth.js

```diff
@@ -1,7 +1,5 @@
 const User = require('./security/user');
 
-const _kuzzle = Symbol();
```

Comment:
What were the advantages of using symbols?

Reply:
To emulate a private property in javascript in a very, very fast way, instead of using Object.defineProperty, which is very slow.
Initially, that solution was applied to our internal Request object, because performances are paramount in that case. I don't really know why it was applied here.

I decided to get rid of any notion of privacy here: if anyone want to play with an undocumented, obviously internal value, well, it's their responsability.